setup.py

```diff
@@ -120,3 +121,12 @@ def get_short_description():
         'Operating System :: OS Independent',
     ],
 )
+
+# remove the line __dev_version__ from k2/python/k2/__init__.py
+with open('k2/python/k2/__init__.py', 'r') as f:
+    lines = f.readlines()
+
+with open('k2/python/k2/__init__.py', 'w') as f:
+    for line in lines:
+        if '__dev_version__' not in line:
```

Comment:
The added entry is removed so that it does not exist in the source tree,
which means it is always synchronized with the latest commit automatically.